core/prisma/schema.prisma

```diff
@@ -280,19 +287,90 @@ enum MemberRole {
 }
 
 model MemberGroup {
-  id          String     @id @default(dbgenerated("gen_random_uuid()"))
-  role        MemberRole @default(editor)
-  community   Community  @relation(fields: [communityId], references: [id], onDelete: Cascade)
+  id          String    @id @default(dbgenerated("gen_random_uuid()"))
+  community   Community @relation(fields: [communityId], references: [id], onDelete: Cascade)
   communityId String
-  createdAt   DateTime   @default(now())
-  updatedAt   DateTime   @default(now()) @updatedAt
+  createdAt   DateTime  @default(now())
+  updatedAt   DateTime  @default(now()) @updatedAt
 
-  permissions Permission[]
-  users       User[]
+  permissions          Permission[]
+  users                User[]
+  formMemberships      FormMembership[]
+  communityMemberships CommunityMembership[]
+  pubMemberships       PubMembership[]
+  stageMemberships     StageMembership[]
 
   @@map(name: "member_groups")
 }
 
+model CommunityMembership {
+  id            String       @id @default(dbgenerated("gen_random_uuid()"))
```

Comment:
Even though we shouldn't need to use the id ever, we still need one because composite ids don't allow nullable columns and two of the three columns that would make up our composite id (communityId, userId, memberGroupId) are nullable.

Reply:
Alternatively we could make userId and memberGroupId not null, and set them to `""` by default. But that feels worse than an unused column to me

Reply:
I see absolutely no problem with giving them ids